packages/quantic/force-app/examples/main/lwc/configurator/configurator.js

```diff
@@ -12,25 +12,42 @@ export default class Configurator extends LightningElement {
             label: option.label ?? option.attribute,
             description: option.description ?? '',
             defaultValue: option.defaultValue,
+            defaultValueAsText: this.formatDefaultValue(option.defaultValue),
             cyId: `cfg-${option.attribute}`,
         }));
     }
 
+    formatDefaultValue(value) {
+        if (typeof value === 'boolean' || typeof value === 'number' || typeof value === 'string') {
```

Comment:
What do you think of ?
```suggestion
        if (['boolean', 'number', 'string'].includes(typeof value)) {
```